spec/ZkVM.md

```diff
@@ -1669,7 +1669,7 @@ the second-from-the-top is not a [contract](#contract-type).
 
 _contract(P) L R_ **left** → _contract(L)_
 
-1. Pops the right [predicate](#predicate) `R`, then the left [predicate](#predicate) `L`.
+1. Pops the right [predicate](#predicate) `R`, then the left [predicate](#predicate) `L` and a [contract](#contract-type) `contract`.
```

Comment:
if you spec to pop the contract, then we need to put it back on the stack on step 4. In both instructions.

Reply:
Step5 for left says: "Replaces the contract’s predicate with L and leaves the contract on stack." Could we change this to ""Replaces the contract’s predicate with L and puts the contract back on stack." ?

Reply:
Yes, let's do that. Thank you.

Reply:
s/puts the contract/pushes the contract/, I'd like to consistently use "pop", "push" terms for the stack operations.